addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java

```diff
@@ -87,34 +100,61 @@ public String getName() {
 
     @Override
     public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
+        long startTime = System.currentTimeMillis();
```

Comment:
Why? We only log if we go past the other check.

Reply:
Probably a bad copy/paste